### DISCUSSION TITLE
$CURRENT_USER alters to concrete value in Conditions tab

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#8473] [Archive]
At conditions tab I set the rule
```json
{
    "user_created": {
        "_eq": "$CURRENT_USER"
    }
}
```
But it alters after saving to:
```json
{
    "user_created": {
        "_eq": "5fee0d79-e54b-4352-91ba-595142daf7c3"
    }
}
```

Is it a bug?

**Directus 9.0.0.rc-91**

**Comment by rijkvanzanten** [comment]
Yes, see https://github.com/directus/directus/issues/7894. Was fixed in https://github.com/directus/directus/pull/8248 and released in rc.94 👍🏻

**Reply by IlyaPetrovM** [reply]
Thank you!
